modules/auxiliary/gather/zoomeye_search.rb

```diff
@@ -41,8 +40,7 @@ def initialize(info = {})
     )
     register_options(
       [
-        OptString.new('USERNAME', [true, 'The ZoomEye username']),
-        OptString.new('PASSWORD', [true, 'The ZoomEye password']),
+        OptString.new('ZOOMEYE_APIKEY', [true, 'The ZoomEye api key']),
```

Comment:
Why are we removing the login?  If we want to add the API Key as an access vector, that makes sense, but why remove existing functionality?

Reply:
There was changes regarding the api access, among which login has been disabled in favor of a token authentication.